ivadomed/object_detection/utils.py

```diff
@@ -239,12 +239,12 @@ def load_bounding_boxes(object_detection_params, subject_path_list, slice_axis,
     bounding_box_dict = {}
     if object_detection_params is None or object_detection_params['object_detection_path'] is None:
         return bounding_box_dict
-    bounding_box_path = os.path.join(object_detection_params['path_output'], 'bounding_boxes.json')
-    if os.path.exists(bounding_box_path):
-        with open(bounding_box_path, 'r') as fp:
+    bounding_box_path = Path(object_detection_params['path_output'], 'bounding_boxes.json')
+    if bounding_box_path.exists():
+        with bounding_box_path.open(mode='r') as fp:
             bounding_box_dict = json.load(fp)
     elif object_detection_params['object_detection_path'] is not None and \
-            os.path.exists(object_detection_params['object_detection_path']):
+            Path(object_detection_params['object_detection_path']).exists():
```

Comment:
Similar to above. Try to use `.get()` pattern more if you can. Love the `Path.exists()` readability